app/serializers/project_serializer.rb

```diff
@@ -21,12 +21,20 @@ class ProjectSerializer
     subjects_export: { include: false },
     aggregations_export: { include: false }
   can_filter_by :display_name, :slug, :beta_requested, :beta_approved,
-    :launch_requested, :launch_approved, :private, :state
+    :launch_requested, :launch_approved, :private, :state, :live
   can_sort_by :launch_date, :activity, :completeness, :classifiers_count,
     :updated_at, :display_name
 
   def self.page(params = {}, scope = nil, context = {})
-    params["state"] = Project.states[params["state"]] if params.key?("state")
+    if params.key?("state")
+      if Project.states.include?(params["state"])
+        params["state"] = Project.states[params["state"]]
+      else
+        params[params["state"]] = true
```

Comment:
it is, w/o a doubt, a hack. If you'd rather, I can add `if project["state"] == "live"` and only replace `params["state"]` with `params["live"] == true`. Less generalized, but the other booleans are all included in `can_filter_by` anyway.